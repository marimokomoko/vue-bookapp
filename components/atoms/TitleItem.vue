<template>
    <h1><slot></slot></h1>
</template>
<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator'
@Component({})
export default class TitleItem extends Vue {
}
</script>