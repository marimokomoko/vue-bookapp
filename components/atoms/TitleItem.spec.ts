import { createLocalVue, mount, Wrapper } from '@vue/test-utils'
import Vuetify from 'vuetify'
import TitleItem from '@/components/atoms/TitleItem.vue'

let wrapper: Wrapper<Vue>;

describe('画面レイアウト', () => {
    it('スナップショットテスト', () => {
        wrapper = mount(TitleItem, {
            localVue: createLocalVue(),
            vuetify: new Vuetify(),
            stubs: ['router-link']
        })
        expect(wrapper.html()).toMatchSnapshot()
    })
})