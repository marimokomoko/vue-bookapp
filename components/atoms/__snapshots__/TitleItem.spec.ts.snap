// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `"<h1></h1>"`;
